lib/crawler.js

```diff
@@ -440,81 +505,18 @@ Crawler.prototype.discoverResources = function(resourceData,queueItem) {
 	if (!queueItem.protocol)
 		queueItem.protocol = "http";
 
-	if (!crawler.parseHTMLComments) {
+	if (!crawler.parseHTMLComments)
```

Comment:
Couple of small things — I used to use this style, but @XhmikosR submitted a PR that removed it, and I more or less agree that we should standardise on braces now (especially as the project is maturing a little.) Could you add the braces back in?

(I wonder if we could stick jscs in there to make this check automatic.)